src/particles/pusher/PlasmaParticleAdvance.H

```diff
@@ -16,12 +17,13 @@
  * \param[in] do_shift boolean to define if the force terms are shifted
  * \param[in] lev MR level
  * \param[in] bins objects containing indices of plasma particles in each tile
+ * \param[in] laser laser pulse, which affects the plasma via the ponderomotive force
  */
 void
 AdvancePlasmaParticles (PlasmaParticleContainer& plasma, Fields & fields,
                         amrex::Geometry const& gm, const bool temp_slice, const bool do_push,
                         const bool do_update, const bool do_shift, int const lev,
-                        PlasmaBins& bins);
+                        PlasmaBins& bins, Laser& laser);
```

Comment:
```suggestion
                        PlasmaBins& bins, const Laser& laser);
```
Besides, the `PlasmaBins&~ should be `const` too here, see #687.